### ISSUE TITLE
[ERROR]: DRM Authentication failed

### ISSUE THREAD

**Issue by quamt** [anidl/multi-downloader-nx#701]
### Program version

5.0.2

### Operating System

Windows

### Type

GUI

### Service

Crunchyroll

### Command used

NIL

### Show ID

GRE50KV36

### Episode

ALL

### Console Output

```Shell
[ERROR] 403: Forbidden
[ERROR] <!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
<HTML><HEAD><META HTTP-EQUIV="Content-Type" CONTENT="text/html; charset=iso-8859-1">
<TITLE>ERROR: The request could not be satisfied</TITLE>
</HEAD><BODY>
<H1>403 ERROR</H1>
<H2>The request could not be satisfied.</H2>
<HR noshade size="1px">
Request blocked.
We can't connect to the server for this app or website at this time. There might be too much traffic or a configuration error. Try again later, or contact the app or website owner.
<BR clear="all">
If you provide content to customers through CloudFront, you can find steps to troubleshoot and help prevent this error by reviewing the CloudFront documentation.
<BR clear="all">
<HR noshade size="1px">
<PRE>
Generated by cloudfront (CloudFront)
Request ID: xxxxxxxx
</PRE>
<ADDRESS>
</ADDRESS>
</BODY></HTML>
[ERROR] Request to DRM Authentication failed: undefined undefined
[ERROR] Download error: Unable to download episode 5 from GRE50KV36
```

### Additional Information

Happens since today for all downloads.

**Comment by quamt** [comment]
Non `DRM ` dl's are ok.

**Comment by AnimeDL** [comment]
Released version v5.0.3 which fixes it

**Comment by quamt** [comment]
Thank you :-)
